### PR TITLE
Dex behind Apache reverse proxy

### PR DESCRIPTION
**Pull request by treydock**
**Modify this link to include the branch name, and possibly the page this PR modifies**:

https://osc.github.io/ood-documentation-test/dex_uri/

**Add your description here**

### CODE REVIEW

File: source/release-notes/v2.1-release-notes.rst [hunk 1]
```diff
@@ -133,6 +134,16 @@ EL9, Ubuntu 22.04 and Ubuntu 20.04 packages
 
 See :ref:`Install Software <install-software>` for instructions on how to install OnDemand using the new EL9, Ubuntu 22.04 and Ubuntu 20.04 packages.
 
+Dex behind Apache reverse proxy by default
+..........................................
+
+By default Dex now sits behind the Apache reverse proxy.
```
Comment (johrstrom) [inline source/release-notes/v2.1-release-notes.rst:146]: Can you add a note here stressing that Dex users are going to see a behavior change.

Maybe a note that it's a more secure default so if they decide to keep it and proxy and they've opened ports, they can now close them and so on.

Comment (treydock) [reply source/release-notes/v2.1-release-notes.rst:146]: I added a warning to describe the breaking change as well as why and that sites can now close the Dex ports.

File: source/release-notes/v2.1-release-notes.rst [hunk 2]
```diff
@@ -137,6 +137,12 @@ See :ref:`Install Software <install-software>` for instructions on how to instal
 Dex behind Apache reverse proxy by default
 ..........................................
 
+  .. warning::
+
+     Dex behing the Apache reverse proxy is a behavior change from OnDemand 2.0 where the reverse proxy configuration was optional.
```
Comment (johrstrom) [inline source/release-notes/v2.1-release-notes.rst]:
```suggestion
     Dex behind the Apache reverse proxy is a behavior change from OnDemand 2.0 where the reverse proxy configuration was optional.
```

File: source/release-notes/v2.1-release-notes.rst (hunk 1 above)
Comment (johrstrom) [reply source/release-notes/v2.1-release-notes.rst:146]: Thanks!

**Review by johrstrom — APPROVED** [review]
👍

**Pull request merged**